tests/test_api.py

```diff
@@ -14,19 +14,148 @@
 # CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 
-import os
 from unittest import mock
 
+import pytest
+
 from fetchcode import api
 
-@mock.patch('fetchcode.api.requests.get')
-def test_fetch(mock_get):
+
+@mock.patch('fetchcode.api.urlopen')
```

Comment:
Do we really need an "api" module?

Reply:
Can you elaborate a little :sweat_smile:

Reply:
may be you could use a flatter namespace and have your api functions attached directly under fetchcode.__init__.py?